wavefront_pyformance/delta.py

```diff
@@ -62,5 +62,5 @@ class DeltaCounter(meters.Counter):
     every time the value is reported.
     """
 
-    DELTA_PREFIX = '\u2206'
-    ALT_DELTA_PREFIX = '\u0394'
+    DELTA_PREFIX = u"\u2206"
+    ALT_DELTA_PREFIX = u"\u0394"
```

Comment:
Any specific reason for using double quotes? Also, you may put delta characters as comment alongside the codes.